source/_integrations/prusalink.markdown

```diff
@@ -17,10 +17,10 @@ ha_platforms:
 ha_dhcp: true
 ---
 
-The PrusaLink integration allows you to monitor your [Prusa 3D printer](https://www.prusa3d.com) and its progress with your Home Assistant installation. This integration is currently limited to the Prusa Mini.
+The PrusaLink integration allows you to monitor your [Prusa 3D printer](https://www.prusa3d.com) and its progress with your Home Assistant installation. This integration works with Prusa MINI and Prusa MK4.
+This integration requires PrusaLink v2. If you have the Prusa MINI, you need to run firmware 4.4.0 or later.
 
-{% include integrations/config_flow.md %}
 
-To obtain the API key, on your printer navigate to Settings -> Network -> PrusaLink. It will then be shown at the bottom of the screen.
+{% include integrations/config_flow.md %}
 
-This integration requires PrusaLink v2. If you have the Prusa MINI, you need to run firmware 4.4.0 or later.
+To obtain the API key, on your printer, navigate to **Settings** > **Network** > **PrusaLink**. It will then be shown at the bottom of the screen.
```

Comment:
This is a prerequisite for setting up the device. I think it should be listed above doing the configuration.

Reply:
Good point. Thank you for pointing it out.